files/tests/test_migrations.py

```
@@ -0,0 +1,277 @@
+        with connection.cursor() as cursor:
+            for media_id in media_ids[:3]:
+                cursor.execute(
+                    "UPDATE files_media SET state = %s WHERE id = %s",
+                    ['private_verified', media_id]
+                )
+
+        # Run the migration (should not raise any exceptions)
+        try:
+            self.executor.migrate(self.migrate_to)
+            migration_successful = True
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)